src/main/java/Events.java

```diff
@@ -0,0 +1,16 @@
+public class Events extends Task{
+    protected String start;
+    protected String end;
+    public Events(String description, String start, String end) {
+        super(description);
+        this.start = start;
+        this.end = end;
+    }
+    public String getIcon(){
```

Comment:
is this an override from` Task` ? Potentially buggy otherwise as you might call the wrong `getIcon()` i think